diesel_cli/src/print_schema.rs

```diff
@@ -65,6 +77,22 @@ pub fn output_schema<W: io::Write>(
     } else {
         write!(out, "{}", definitions)?;
     }
+
+    if let Some(ref patch_file) = config.patch_file {
+        let output = Command::new("patch")
+            .arg(out_path)
+            .arg(patch_file)
+            .output()?;
```

Comment:
This looks like a place where we could special-case the `unknown command: patch` case and tell people they need to install it

Reply:
I don't know how we could do that other than checking with `which`. But then we have the same problem